common/changes/@rushstack/webpack5-module-minifier-plugin/webpack_2025-12-16-02-42.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Updated Webpack version",
+      "type": "minor",
```

```suggestion
      "comment": "",
      "type": "none",
```

The dependency update doesn't matter here.